docs/specifications/tests/Nameserver-TP/nameserver11.md

```diff
@@ -3,91 +3,142 @@
+   2. If there is no DNS response, then go to next name server.
+   3. Else, if the DNS response has the RCODE "FORMERR" ([IANA RCODE List]), then go to next name server.
+   4. Else, if there is an "OPTION-CODE" present in the response, then add name server IP
+      to the "Unknown Option Code" set and go to next name server.
+   5. Else, if the DNS response meet the following four criteria,
+      then go to next name server (no error):
+      1. The SOA is obtained as response in the ANSWER section.
+      2. If the DNS response has the RCODE "NOERROR" ([IANA RCODE List]).
+      3. The pseudo-section response has an OPT record with version set to 0.
+      4. There is no "OPTION-CODE" present in the response.
+
+5. If the "Unknown Option Code" set is non-empty, then output
+   *[NS11_UNKNOWN_OPTION_CODE]* with the name servers IP addresses from the set.
+
+
 ## Outcome(s)
 
 The outcome of this Test Case is "fail" if there is at least one message
-with the severity level *ERROR* or *CRITICAL*.
+with the severity level *[ERROR]* or *[CRITICAL]*.
 
 The outcome of this Test Case is "warning" if there is at least one message
-with the severity level *WARNING*, but no message with severity level
+with the severity level *[WARNING]*, but no message with severity level
 *ERROR* or *CRITICAL*.
 
-The outcome of this Test case is "pass" in all other cases.
-
-Message                           | Default severity level (when message is outputed)
-:---------------------------------|:--------------------------------------------------
-NO_RESPONSE                       | DEBUG
-NO_EDNS_SUPPORT                   | WARNING
-NS_ERROR                          | WARNING
-UNKNOWN_OPTION_CODE               | WARNING
+In other cases, no message or only messages with severity level
+*[INFO]* or *[NOTICE]*, the outcome of this Test Case is "pass".
 
 ## Special procedural requirements
 
-If either IPv4 or IPv6 transport is disabled, ignore the evaluation of the
-result of any test using this transport protocol and log a message reporting
-the ignored result.
+If either IPv4 or IPv6 transport is disabled, skip sending queries over that
+transport protocol. A message will be outputted reporting that the transport
+protocol has been skipped.
 
 ## Intercase dependencies
 
 None.
 
-
-
-[Basic04]:               ../Basic-TP/basic04.md
-[IANA-DNSSYSTEM-PARAMETERS]: https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#dns-parameters-11
-[Method4]: ../Methods.md#method-4-obtain-glue-address-records-from-parent
-[Method5]: ../Methods.md#method-5-obtain-the-name-server-address-records-from-child
-[NO_EDNS_SUPPORT]: #outcomes
-[NO_RESPONSE]: #outcomes
-[NS_ERROR]: #outcomes
-[RFC 6891, section 6.1.2]: https://tools.ietf.org/html/rfc6891#section-6.1.2
-[RFC 6891]: https://tools.ietf.org/html/rfc6891
-[UNKNOWN_OPTION_CODE]: #outcomes
+## Terminology
+
+No special terminology for this test case.
+
+[NS11_UNKNOWN_OPTION_CODE]:             #summary
+
```

Comment:
Please put that in the right place in the list of links.

Reply:
Fixed